lib/mapCache.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+module.exports = function createMapCache (fetch, fallback) {

```

Comment:
can you add a comment on top explaining what this does?

Reply:
I've added a comment explaining what it does + an example - let me know if you'd prefer a test or if the documentation is unclear.